lib/widgets/post_modal.dart

```diff
@@ -99,6 +102,7 @@ class PostBottomModal extends StatelessWidget {
                             child: const Text("Yes"),
                           ),
                           TextButton(
+                            key: const Key('no'),
                             onPressed: () {
                               Navigator.pop(context);
                             },
```

Comment:
These lines are missing. Please write the test for it.

Reply:
https://app.codecov.io/gh/PalisadoesFoundation/talawa/pull/2304/blob/lib/widgets/post_modal.dart

Reply:
I have written test for these lines also